src/main/java/com/amazon/opendistroforelasticsearch/ad/model/AnomalyDetectorExecutionInput.java

```diff
@@ -31,6 +31,7 @@
 import java.io.IOException;
 import java.time.Instant;
 
+import org.apache.logging.log4j.util.Strings;
```

Comment:
Are we using the log4j util.Strings for not empty check? I guess it suppose to be org.opensearch.common.Strings?

Reply:
Good catch, will fix it